Source/SuperLinq/Collections/UpdatablePriorityQueue.cs

```diff
@@ -650,12 +653,16 @@ public void EnqueueRangeMinimum(IEnumerable<TElement> elements, TPriority priori
 	/// </summary>
 	public void Clear()
 	{
+#if !NETSTANDARD
```

Comment:
Do you have a preference over how to address the code style issues here? The `#if` makes the resulting indentation incorrect.